src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1107,10 +1111,11 @@ void cbmc_parse_optionst::help()
     " --yices                      use Yices\n"
     " --z3                         use Z3\n"
     " --refine                     use refinement procedure (experimental)\n"
-    " --refine-strings             use string refinement (experimental)\n"
+    " --refine-strings             use string refinement\n"
```

Comment:
I would not remove the `(experimental)` mention, as string support is not fit for bmc verification. We've been developing it in the prospect of test generation, which is very different. What do you think?